src/containers/IntentService.jsx

```diff
@@ -0,0 +1,111 @@
+/* global cozy */
+
+import styles from '../styles/services'
+
+import React, { Component } from 'react'
+
+import Loading from '../components/Loading'
+import ServiceBar from '../components/services/ServiceBar'
+import ViewerService from '../components/services/ViewerService'
+
+export default class IntentService extends Component {
```

Comment:
I guess we could write something simpler than this very big and not designed component.